webapp/src/js/router.js

```diff
@@ -119,6 +124,7 @@
         views: {
           content: {
             controller: 'AnalyticsTargetsCtrl',
+            controllerAs: '$ctrl',
```

Comment:
There's some inconsistent naming going on here.

Reply:
There is indeed, I'm not sure what to do about it really, ideally I would name everything `$ctrl` as when we move everything to components they'll all default to `$ctrl` and everything will work fine (I think). In the current structure though if everything is named `$ctrl` then you get bugs because of nested templates. I guess the most sensible thing to do for now would just make everything fit the style of
```
controller: 'AnalyticsTargetsCtrl',
controllerAs: 'analyticsTargetsCtrl'
```
?

Reply:
Yeah that works for me. It'd be easy to do a find/replace to `$ctrl` later...

Reply:
Sounds good will change